### PR TITLE
refactor(userinforfetcher tests): simplify keycloak setup

### PR DESCRIPTION
**Pull request by fhennig**
# Description

WIP

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

**Review by sbernauer — CHANGES_REQUESTED** [review]
LGTM otherwise!

File: tests/templates/kuttl/keycloak-user-info/04-keycloak-realm-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        "username" : "alice",
+        "enabled" : true,
+        "emailVerified" : true,
+        "firstName" : "Alice",
+        "lastName" : "Adams",
+        "email" : "alice@example.org",
+        "credentials" : [ {
+          "type" : "password",
+          "userLabel" : "My password",
+          "secretData" : "{\"value\":\"hogrQRLTAPBws9RgxZF/d3+EvPvUc7AN1egnmMnuWBQ=\",\"salt\":\"DYkgbXwZ2uhvJ+k94Xr7lg==\",\"additionalParameters\":{}}",
```
[the remark this answers is not in the document]
Comment (fhennig) [reply tests/templates/kuttl/keycloak-user-info/04-keycloak-realm-cm.yaml:33]: not possible

**Review by sbernauer — APPROVED** [review]
LGTM when tests pass

**Comment by fhennig** [comment]
jenkins run: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/opa-operator-it-custom/88/

**Pull request merged**